index.bs

```diff
@@ -3261,6 +3271,49 @@ To <dfn noexport>set an OS-support header</dfn> given a [=header list=]
 1. [=header list/Set a structured field value=] given
     ("<code>[=Attribution-Reporting-Support=]</code>", |dict|) in |headers|.
 
+# User Agent Automation # {#automation}
+
+For the purposes of user agent automation and website testing, this document
+defines the below [[WebDriver]] [=extension commands=] to control the API
+configuration.
+
+## Set debug mode enabled ## {#webdriver-setdebugmodeenabled}
+
+<figure id="table-webdriver-setdebugmodeenabled" class="table">
+    <table class="data">
+        <thead>
+            <tr>
+                <th>HTTP Method</th>
+                <th>URI Template</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>POST</td>
+                <td>`/session/{session id}/ara/`<dfn>`setdebugmodeenabled`</dfn></td>
```

Comment:
Based on other commands in the WebDriver spec, maybe the final component of the URI template should be `debugmode` rather than `setdebugmodeenabled`, i.e. a noun rather than a command?